### ISSUE TITLE
Table where rows and columns are two different relationships

### ISSUE THREAD

**Issue by hiszpek** [saltcorn/saltcorn#1663]
I want to implement a table where rows and columns are two different relationships, similar to **Pivot table** view pattern, but where the cells contain a value from the many-to-many relationship.

Example:

I have four tables:

`groups`:
- `id`
- `name`

`members`:
- `id`
- `name`
- `group_id`

`meetings`:
- `id`
- `date`
- `group_id`

`meetings_members`:
- `id`
- `meeting_id`
- `member_id`
- `attendance`

I want to generate the following table:

|  | Meeting 1 | Meeting 2 | Meeting 3 |
| --- | --- | --- | --- |
| Member 1 | 1 | 0 | 0 |
| Member 2 | 0 | 1 | 1 |
| Member 3 | 0 | 1 | 0 |

Where the `1` and `0` values in the cells are from the `meetings_member.attendance` value, and the members and meetings belong to a given group (passed in the URL, like the **Show** view pattern).

Is this possible with Saltcorn's features? Or is there a plugin that I can use?

**Comment by glutamate** [comment]
yes there is a very powerful plugin called pivottable. Install that and create a Pivot table on the join table `meetings_members` - then you get a little UI based on https://pivottable.js.org/examples/ that should feel a bit like excel. Does this work for you?

**Comment by hiszpek** [comment]
@glutamate Thank you very much for the fast response.

I tried using the Pivot table view pattern, but couldn't achieve a result where the cells are a value from the `meetings_members` table (I want them to be populated with the `attendance` column's value). 

I created a Pivot table with members names on the left and meeting ids on the top, but I can only select one of the aggregating values, like *Count* here:
![image](https://user-images.githubusercontent.com/74194396/220151962-19b837e2-6c83-422d-9bd1-5288c7aee7be.png)

Am I missing something in the configuration? Or does it need some additional work, like a plugin?
